tests/phpunit/integration/Modules/Sign_In_With_GoogleTest.php

```diff
@@ -121,12 +116,6 @@ public function test_render_signinwithgoogle() {
 		$this->assertStringContainsString( sprintf( '"theme":"%s"', Sign_In_With_Google_Settings::THEME_LIGHT['value'] ), $output );
 		$this->assertStringContainsString( sprintf( '"shape":"%s"', Sign_In_With_Google_Settings::SHAPE_RECTANGULAR['value'] ), $output );
 
-		// If WooCommerce is not active, the button should not be rendered in
-		// the WooCommerce form.
-		do_action( 'woocommerce_login_form_start' );
```

Comment:
Firing this action without capturing output caused it to leak into the terminal.

It's anyway been removed since this action should only be run when WC is active.